Makefile

```diff
@@ -53,11 +53,11 @@ else
 endif
 
 export TARGET_OUT ?= $(shell pwd)/out/$(TARGET_OS)_$(TARGET_ARCH)
-export TARGET_OUT_LINUX ?= $(shell pwd)/out/linux_amd64
+export TARGET_OUT_LINUX ?= $(shell pwd)/out/linux_$(TARGET_ARCH)
```

Comment:
Top-level makefile changes should be changed in https://github.com/istio/common-files. This change would be overwritten in any update of the common-files repository.

Also this change is incorrect. TARGET_OUT_LINUX should be generalized - see https://github.com/istio/istio/blob/master/Makefile.core.mk#L290-L293 for directional guidance on what needs to be done here.